bin/executor

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
i think this should no longer be here

Reply:
I think it does, it runs php file via shell

Reply:
I mean the I think the whole `executor` was moved to open-runtimes, @Meldiron knows better

Reply:
removed